apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerSettingsPartial.cs

```diff
@@ -34,4 +35,14 @@ public Logger Logger
         get => _logger;
         set => _logger = GaxPreconditions.CheckNotNull(value, nameof(value));
     }
+
+    /// <summary>
+    /// Specifies whether leader routing is enabled or not. This is true by default.
+    /// </summary>
+    /// <remarks>
+    /// If this value is true some operations will always be explicitly routed to the leader,
+    /// some operations will never be explicitly routed to the leader, and some operations will
+    /// be routed to the leader depending on the transaction type they are using.
+    /// </remarks>
+    internal bool LeaderRoutingEnabled { get; set; } = true;
```

Comment:
I'm slightly surprised this is internal rather than public. Is there a reason for that?